packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -392,70 +398,76 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
   }
 
   /**
-   * Requests attestations for a phone number
-   * @param phoneNumber The phone number for which to request attestations for
+   * Requests a new attestation
+   * @param identifier Attestation identifier (e.g. phone hash)
    * @param attestationsRequested The number of attestations to request
    */
-  async request(phoneNumber: string, attestationsRequested: number) {
-    const phoneHash = PhoneNumberUtils.getPhoneHash(phoneNumber)
+  async request(identifier: string, attestationsRequested: number) {
     const tokenAddress = await this.kit.registry.addressFor(CeloContract.StableToken)
     return toTransactionObject(
       this.kit,
-      this.contract.methods.request(phoneHash, attestationsRequested, tokenAddress)
+      this.contract.methods.request(identifier, attestationsRequested, tokenAddress)
     )
   }
 
   /**
    * Selects the issuers for previously requested attestations for a phone number
-   * @param phoneNumber The phone number for which to request attestations for
+   * @param identifier Attestation identifier (e.g. phone hash)
    */
-  selectIssuers(phoneNumber: string) {
-    const phoneHash = PhoneNumberUtils.getPhoneHash(phoneNumber)
-    return toTransactionObject(this.kit, this.contract.methods.selectIssuers(phoneHash))
+  selectIssuers(identifier: string) {
+    return toTransactionObject(this.kit, this.contract.methods.selectIssuers(identifier))
   }
 
   revealPhoneNumberToIssuer(
     phoneNumber: string,
     account: Address,
     issuer: Address,
-    serviceURL: string
+    serviceURL: string,
+    salt?: string
   ) {
+    const body: AttesationServiceRevealRequest = {
```

Comment:
Would it be not fine to just pass `salt,` even if it is undefined?

Reply:
Just tested, you're right. Will clean this up